packages/jest/README.md

```diff
@@ -154,10 +154,14 @@ Automatic checks can also be enabled using environment variables
 SA11Y_AUTO=1 SA11Y_CLEANUP=1 jest
 ```
 
--   Invoking `jest` with environment variables as above will enable automatic checks with no changes required to `setup()`
--   The environment variables can be used to set up parallel builds e.g., in a CI environment without code changes to `setup()` to opt-in to automatic checks
--   Setting `SA11Y_DEBUG=1` will output verbose logging
--   `SA11Y_AUTO_ONLY` can be used to specify a comma seperated list of files to run automatic checks on while other files are ignored.
+- Invoking `jest` with environment variables as above will enable automatic checks with no changes required to `setup()`
+- The environment variables can be used to set up parallel builds e.g., in a CI environment without code changes to `setup()` to opt-in to automatic checks
+- Setting `SA11Y_DEBUG=1` will output verbose logging
+- `SA11Y_AUTO_FILTER` can be used to specify a comma seperated list of test file paths to filter for automatic checks
+  - When specified, automatic checks will be run only on given files
+  - Files can be excluded from automatic checks by prefixing the filename with `!`
```

Comment:
Do you need a ! in front of each excluded test, or just one in front of the comma-separated list?

Reply:
Good point, changed doc to make it explicit that ! is needed for each file